uvi_web/templates/welcome_frame.html

```diff
@@ -21,6 +21,6 @@ <h1>Welcome to the Unified Verb Index</h1>
 Any opinions, findings, and conclusions or recommendations expressed in this material are those of the authors and do not necessarily reflect the views of the National Science Foundation.</p>
 
 <!-- <button class='btn' id='fade_welcome_button'>Take Me to the UVI!</button> -->
-<button class='btn' id='fade_welcome_button'>Take Me to the UVI!</button>
+<button class='btn btn-light btn-lg' onclick="window.location.href='/uvi_search'">Take Me to the UVI!</button>
```

Comment:
Clicking on "Take me to the UVI" will no longer use search.html. Upcoming commits will be used to remove the html from the project entirely.